code/unclassified/lapindrom_checker/lapindrome_checker.py

```diff
@@ -0,0 +1,26 @@
+# part of cosmos by opengenus foundation
+
+from __future__ import division, print_function  # if python 2.x used
```

Comment:
We are moving toward's Python 3, so don't need this. You don't need print_function for Python2 either, because your print statement only uses parentheses.